pkgs/development/libraries/science/astronomy/indilib/indi-3rdparty.nix

```diff
@@ -13,10 +13,14 @@
 , indilib
 , libgphoto2
 , libraw
+, libraw1394
 , libftdi1
 , libdc1394
 , gpsd
 , ffmpeg
+, udev
+, pkg-config
```

Comment:
While the cmake file can use pkg-config, it is not required because all libraries are still found via CMake. `pkg-config` may be removed.

Reply:
Has been removed